### PR TITLE
Create remove policy assignment article

### PR DESCRIPTION
**Pull request by viv-liu**

### CODE REVIEW

File: api-reference/beta/api/policy_remove_assignment [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# Remove policy assignment
```
Comment (PatAltimore) [inline api-reference/beta/api/policy_remove_assignment]: Rename the file to include the .md extension.

File: api-reference/beta/api/policy_remove_assignment.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+# Remove policy assignment
```
Comment (dkershaw10) [inline api-reference/beta/api/policy_remove_assignment.md]: This is good, but the https://developer.microsoft.com/en-us/graph/docs/api-reference/beta/resources/policy topic (methods table) needs to be updated with a link to this topic.

**Comment by jasonjoh** [comment]
@dkershaw10 @viv-liu This one's been lingering for over a year now. Can we merge this, assuming we add a link to the resource topic's method table?

**Comment by viv-liu** [comment]
@jasonjoh please ping Dan directly. I have no idea if this doc is up to date or relevant any more as I am on a different team.

**Pull request closed without merging**